paddle/framework/attribute.h

```diff
@@ -45,7 +45,19 @@ class LargerThanChecker {
  public:
   explicit LargerThanChecker(T lower_bound) : lower_bound_(lower_bound) {}
   void operator()(T& value) const {
-    PADDLE_ENFORCE(value > lower_bound_, "larger_than check fail");
+    PADDLE_ENFORCE(value > lower_bound_, "larger_than check fails.");
+  }
+
+ private:
+  T lower_bound_;
+};
+
+template <typename T>
+class EqualLargerThanChecker {
```

Comment:
The name is better to compatible with gtest. Such as `CHECK_GE` or something?

Reply:
`EqualLargerThan` is a function, not a macro, so the name shall not be too short.